app/views/text_components/_form.html.haml

```diff
@@ -23,63 +23,53 @@
-            Ingredients
-          %button.btn.btn-primary.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :type => "button"}
-            %span.caret
-            %span.sr-only
-          %ul.dropdown-menu
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-picture{"aria-hidden" => "true"}
-                Picture
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-facetime-video{"aria-hidden" => "true"}
-                Video
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-play{"aria-hidden" => "true"}
-                Audio
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-link{"aria-hidden" => "true"}
-                Link
-            %li.divider{:role => "separator"}
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-tag{"aria-hidden" => "true"}
-                Markup 1
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-tag{"aria-hidden" => "true"}
-                Markup 2
-            %li
-              %a{:href => "#"}
-                %span.glyphicon.glyphicon-tag{"aria-hidden" => "true"}
-                Markup 3
-    %div.seamless-textarea.text-component-input-fields
-      = f.input :heading, as: :text, placeholder: 'Title', label: false
-      = f.input :introduction, as: :text, placeholder: 'Introduction', label: false, input_html: { class: 'introduction' }
-      = f.input :main_part, as: :text, placeholder: 'Main part', label: false, input_html: { rows: 10 }
-      = f.input :closing, as: :text, placeholder: 'Closing', label: false
-    .row
-      .col-sm-12
-        %h2 Questions/Answers for Chatfuel
-        .container-fluid
-          = f.simple_fields_for :question_answers do |question_answer|
-            = render 'question_answers/fields', f: question_answer
-          .links
-            .row
-              = link_to_add_association button_tag("Add a question and an answer", type: 'button', class: 'btn btn-primary'), f, :question_answers, partial: 'question_answers/fields.html.haml'
+    %h2 Text
+    %p.help Please write the text here. To add dynamic fields, like temperature values via toolbar.
+    .text-editor
+      .text-editor__toolbar
+        %ul.list-inline
+          %li
+            .text-editor__toolbar__item
+              %span.glyphicon.glyphicon-picture{"aria-label" => "Picture"}
+          %li
+            .text-editor__toolbar__item
+              %span.glyphicon.glyphicon-facetime-video{"aria-label" => "Video"}
+          %li
+            .text-editor__toolbar__item
+              %span.glyphicon.glyphicon-play{"aria-label" => "Audio"}
+          %li
+            .text-editor__toolbar__item
+              %span.glyphicon.glyphicon-link{"aria-label" => "Link"}
```

Comment:
@drjakob can we delete buttons that have no effect for now? For the record: We had these buttons to comply to our design template. I think it adds confusion to display not implemented functionality.